docs/cloudfoundry/config_examples/manifest.yaml

```diff
@@ -66,25 +66,42 @@ applications:
 
       # Include the section below to use the cf-bigip-ctlr as a Service Broker
       # THE SETTINGS IN THIS SECTION ARE ROUTE-SPECIFIC
+      # The first health monitor references a monitor that already exists in the Common partition on the BIG-IP device
+      # The second health monitor creates a new monitor on the BIG-IP device.
+      # Referenced health monitors should be created or referenced from the Common partition.
```

Comment:
Replace with 
"See http://clouddocs.f5.com/containers/latest/cloudfoundry/cf-per-route-virtuals.html for more information"

Reply:
Because we've removed the comments explaining what health monitors you can use, change the text in the Add BIG-IP Health Monitors section of `/cloudfoundry/cf-per-route-virtuals.rst` as follows:

```
You can attach an existing health monitor, create a new one, or both:

- use any health monitor that exists in the :code:`/Common` partition on the BIG-IP system 
- create a new health monitor in the partition the |cfctlr| manages (in this case, "cf")
```

Reply:
Is it weird to include the docs link in the example manifest when that file is included in the doc we're referencing? Up for debate. I thought it could be helpful for users that find the YAML file via GitHub or other means.

Reply:
Ya I think even though it looks a little weird who knows where these manifests might end up so its good to have a reference to where it originated from and where more info can be found.